pkg/kubelet/rkt/rkt.go

```diff
@@ -937,15 +985,21 @@ func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *k
 		newUnitOption("Service", "ExecStopPost", markPodFinished),
 		// This enables graceful stop.
 		newUnitOption("Service", "KillMode", "mixed"),
+		// Track pod info for garbage collection
+		newUnitOption(unitKubernetesSection, unitPodUID, string(pod.UID)),
```

Comment:
@dcbw I mean we can read the podmanifest in the response of the ListPods() or InspectPod() from rkt api service.
OOB discussion with @euank, I agree that this can be more user-friendly for debugging. But for developers who will work on the code, I would say a single source of truth would be easy to manage.
However, I won't block the PR on this.